closed/src/java.base/share/classes/sun/security/provider/NativeDigest.java

```diff
@@ -24,7 +24,7 @@
  */
 /*
  * ===========================================================================
- * (c) Copyright IBM Corp. 2018, 2019 All Rights Reserved
+ * (c) Copyright IBM Corp. 2018, 2023 All Rights Reserved
```

Comment:
While you're here, could you please delete the extra blank line (one of lines 30 or 31)?

Reply:
Updated in recent commit to remove one of the duplicate blank lines.